px-spinner.html

```diff
@@ -76,34 +79,6 @@
                 return Px.Modernizr.smil;
               }
             }
-        },
-        /**
-         *
-         * This method checks whether the finished attribute is true,
-         * and if it is, hides the element. if it's false, it shows it
-         *
-         */
-        _finishedChanged: function(newValue, oldValue) {
```

Comment:
And also the `_finishedChanged` observer should be removed from the `finished `property definition.